---
xbee_address:
  base_station: 41AF1B21
  boat1: 41AF1BAD
  boat2: 41AF1B4B 
  boat3: 417E5F98
  boat4: 41AF1C55
  boat5: 417E6274
  boat6: 417E628A
  boat7: 41AF1B41
